platform/src/main/java/org/stellar/anchor/platform/config/PropertySecretConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.stellar.anchor.platform.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.stellar.anchor.config.SecretConfig;
+
+@Data
+public class PropertySecretConfig implements SecretConfig {
+  @Value("${secret.sep10.jwt_secret:#{null}}")
+  private String jwtSecretKey;
```

Comment:
💡  Can you rename `jwtSecretKey` to something that refers to SEP-10, since it's strongly tied with the SEP? Maybe `sep10JwtSecret`?

Reply:
Oh yeah, that's right. We should prefix `sep10` to make it consistent with the signing seed!